test/jest/fixtures.js

```diff
@@ -122,14 +123,24 @@ module.exports = function registerFixtures(global) {
   
   global.registerWorkerFixture('browser', async ({browserType, defaultBrowserOptions}, test) => {
     const browser = await browserType.launch(defaultBrowserOptions);
-    await test(browser);
-    await browser.close();
+    try {
+      await test(browser);
```

Comment:
test can throw? I thought it catches internally?

Reply:
I never made it catch and now I can use it...